server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeInternalRPCServiceImpl.java

```diff
@@ -137,8 +146,11 @@ public class DataNodeInternalRPCServiceImpl implements IDataNodeRPCService.Iface
   private final SchemaEngine schemaEngine = SchemaEngine.getInstance();
   private final StorageEngineV2 storageEngine = StorageEngineV2.getInstance();
 
+  private final RegionManager regionManager;
+
   public DataNodeInternalRPCServiceImpl() {
     super();
+    regionManager = new RegionManager(schemaEngine, storageEngine);
```

Comment:
We need to confirm that the RegionManager is shared in different instances of DataNodeInternalRPCServiceImpl

Reply:
DataNodeInternalRPCServiceImpl is a global unique instance and has the same life cycle as IoTDB.